<!doctype html>
<html>
<head>
	<title>Documentation for INDELible &ndash; A powerful and flexible simulator of biological evolution.</title>
</head>
<body>
	<h1>INDELible</h1>
	<p>INDELible is a new, portable, and flexible application for biological sequence simulation that combines many features in the same place for the first time. Using a length-dependent model of indel formation it can simulate evolution of multi-partitioned nucleotide, amino-acid, or codon data sets through the processes of insertion, deletion, and substitution in continuous time.<br>

	Nucleotide simulations may use the general unrestricted model or the general time reversible model and its derivatives, and amino-acid simulations can be conducted using fifteen different empirical rate matrices. Substitution rate heterogeneity can be modelled via the continuous and discrete gamma distributions, with or without a proportion of invariant sites. INDELible can also simulate under non-homogenous and non-stationary conditions where evolutionary models are permitted to change across a phylogeny.<br>

	Unique among indel simulation programs, INDELible offers the ability to simulate using codon models that exhibit nonsynonymous/synonymous rate ratio heterogeneity among sites and/or lineages.</p>
	<hr>
	<p>The model parameters are read from a file called <tt>control.txt</tt>. The following two sections explain the different parameters and their effect on the generated sequences.</p>
	<ul>
		<li><a href=examples.shtml>Tutorials and Examples</a></li>
		<li><a href=manual.shtml>Manual</a></li>
	</ul>
</body>
</html>
